src/onnx/parse_instancenorm.cpp

```diff
@@ -52,6 +52,7 @@ struct parse_instancenorm : op_parser<parse_instancenorm>
         auto scale = args[1];
         auto bias  = args[2];
         auto dims  = x->get_shape().lens();
+        auto dtype = x->get_shape().type();
```

Comment:
you need to also check if the type is valid, similar to what we do in the parse_random ops:
```
const std::set<shape::type_t> valid_types = {
        shape::float_type, shape::half_type, shape::double_type};
...
if(not contains(valid_types, dtype))
            MIGRAPHX_THROW(opd.op_name + ": invalid input type: " + std::to_string(dtype) +
                           ". Valid types are float, half, or double.");
```

Reply:
Added. Thanks.